docs/candy-kata/index.html

```diff
@@ -0,0 +1,97 @@
+
+<!DOCTYPE html>
+<!--
+  ~ Copyright (c) 2017 Goldman Sachs.
```

Comment:
You should update this copyright to say "2021 Goldman Sachs and others."

Reply:
Updated